commands/version/index.js

```diff
@@ -650,13 +662,17 @@ class VersionCommand extends Command {
       : tag;
 
     return Promise.resolve()
-      .then(() => gitCommit(message, this.gitOpts, this.execOpts))
-      .then(() => gitTag(tag, this.gitOpts, this.execOpts))
+      .then(() => this.shouldCommit && gitCommit(message, this.gitOpts, this.execOpts))
+      .then(() => this.shouldTag && gitTag(tag, this.gitOpts, this.execOpts))
       .then(() => [tag]);
   }
 
   gitPushToRemote() {
-    this.logger.info("git", "Pushing tags...");
+    if (this.shouldTag) {
```

Comment:
Since `gitPushToRemote()` method is called when either `this.shouldTag` or `this.shouldCommit` is true, if inside the method, and `!this.shouldTag`, means only `this.shouldCommit` equals `true`. Thus, it's safe to fallback to log the commit message.